### PR TITLE
F#7 rectangular tables

### PR DESCRIPTION
**Pull request by deathtanks**

### CODE REVIEW

File: robert_gazebo/models/table_rectangular_low/model.sdf [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            <uri>model://robert_gazebo/meshes/table_rectangular_low_visual.dae</uri>
+            <scale>1 1 1</scale>
+          </mesh>
+         </geometry>
+      </visual>
+
+      <collision name="collision">
+        <geometry>
+          <mesh>
+            <uri>model://robert_gazebo/meshes/table_rectangular_low_collisium.dae</uri>
```
Comment (AndriyPt) [inline robert_gazebo/models/table_rectangular_low/model.sdf]: wrong DAE file name :)

**Review by AndriyPt — CHANGES_REQUESTED** [review]

**Comment by AndriyPt** [comment]
Please fix small issue of file name and happy to merge :)

**Comment by sofiia-tesliuk** [comment]
Problem solved, everything is ok.

**Comment by AndriyPt** [comment]
@sophia-tslk @deathtanks  Please add `.DS_Store` to .gitignore and delete these files from this branch :)

**Comment by AndriyPt** [comment]
please remove files 
```
robert/robert_gazebo/models/puffy/puffy.config
robert/robert_gazebo/models/puffy/puffy.sdf
```
from this branch :)
Please also restore file `robert/robert_gazebo/worlds/ucu.world`

**Comment by sofiia-tesliuk** [comment]
done.

**Comment by v-lavrushko** [comment]
Done.

**Review by AndriyPt — APPROVED** [review]

**Pull request merged**